internal/pkg/describe/webapp.go

```diff
@@ -205,6 +198,25 @@ func (d *WebAppDescriber) ECSParams(envName string) (*WebAppECSParams, error) {
 	}, nil
 }
 
+// GetServiceArn returns the ECS service ARN of the application in an environment.
+func (d *WebAppDescriber) GetServiceArn(envName string) (*ServiceArn, error) {
```

Comment:
Eventually - we should put these in a struct that's like ServiceDescriber - so we can re use all this for other service types.

Reply:
Yeah I agree! A good idea to have `ServiceDescriber` that can be shared across all app type. I'd say this requires more design and we can also know the requirement for the struct design better once we have other app types. What do you think?